neutromeratio/equilibrium.py

```diff
@@ -8,13 +8,14 @@
 
 class LangevinDynamics(object):
```

Comment:
I think this would be simpler if the energy_and_force here is given as a function only of x (rather than as an object that contains an attribute that's a function of both x and lambda).

So in the context of a free energy calculation that uses LangevinDynamics, you would do something like:
```python
atoms, x0, temperature = ...
ani = ANI1_force_and_energy( ... )

for lam in [0.0, 0.5, 1.0]:
    energy_and_force = lambda x : ani.calculate_force(x, lam)
    traj, energy_traj = LangeinDynamics(atoms, temperature, energy_and_force).run_dynamics(x0, n_steps=100, ...)
```

Reply:
agree! put on the TODO list!